app/common/translate.py

```diff
@@ -220,13 +274,34 @@ def sanitize_and_translate(self, text: str, wrap_width: int, max_lines=None, add
         for alignment in alignments:
             output = output.replace(alignment, "")
 
+        # trim multiple ellipses to a single one
+        ellipses = [
```

Comment:
Each of these lines needs to be indented by 4 spaces.

Reply:
Right, mb